packages/app/src/server/models/bookmark-folder.ts

```diff
@@ -112,5 +139,19 @@ bookmarkFolderSchema.statics.updateBookmarkFolder = async function(bookmarkFolde
 
 };
 
+bookmarkFolderSchema.statics.insertOrUpdateBookmarkedPage = async function(pageId: IPageHasId, userId: Types.ObjectId | string, folderId: string):
+Promise<BookmarkFolderDocument | null> {
```

Comment:
Is there a possibility of null?

Reply:
I think it's impossible to return null , so I removed null return type